hass_nabucasa/iot_base.py

```diff
@@ -75,6 +75,11 @@ def package_name(self) -> str:
         """Return package name for logging."""
         raise NotImplementedError
 
+    @property
+    def ws_heartbeat(self) -> float | None:
```

Comment:
Only the CloudIoT class can use this for now, so made it a property (like ws_server_url) to allow those who inherit this base class to override it, with a default to `None` (current behaviour).